lisa_commands/commands.py

```diff
@@ -284,14 +282,14 @@ def kill_me(_query, slack_event):
             'By order of the SDF Armed Forces you have been killed',
             'Ask not for whom the bell tolls. It tolls for thee ~ John Donne',
             "We all have but one life to live. Well you do. I'm a bot, and I have backups",
-            ('A coward dies a thousand times before thier death, but the valiant',
+            ('A coward dies a thousand times before thier death, but the valiant'
```

Comment:
nit: When using paren notation for long strings, it can be easy to lose track of some whitespace. We'll need a space after valiant here, otherwise valianttaste will be a great new word for the English lexicon. :D

Double-checking other paren-notation strings for this would be good as well.